server/config/config.js

```diff
@@ -23,6 +23,16 @@ const config = {
     dialect: 'postgres',
     secret: process.env.JWT_SECRET
   },
+  facebook: {
```

Comment:
nope. lets leave our environment config as is. You can create a new file for facebook and google. Sorry I wasn't clear

Reply:
Ok... I will effect this change now.